src/core/border_router/routing_manager.hpp

```diff
@@ -240,9 +240,6 @@ class RoutingManager : public InstanceLocator
                              otRoutePreference  aRoutePreference = OT_ROUTE_PREFERENCE_MED);
     bool NetworkDataContainsOmrPrefix(const Ip6::Prefix &aPrefix) const;
 
-    // Decides the first prefix is numerically smaller than the second one.
-    static bool     IsPrefixSmallerThan(const Ip6::Prefix &aFirstPrefix, const Ip6::Prefix &aSecondPrefix);
-    static bool     IsValidOmrPrefix(const NetworkData::OnMeshPrefixConfig &aOnMeshPrefixConfig);
```

Comment:
@abtink This should be a mistake :)

The CI is failing because of this.

Reply:
Yes. Thanks.   :)
I rebased on `main` and I think this was then removed by mistake.

Reply:
It's now passing all tests.